web/src/components/card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Card() {
+    return (
+        <div className="absolute bg-white left-24 rounded-bl-lg top-44 w-5/12">
+            <h1 className="pl-6 pt-7 mb-6 text-brand-1 text-4xl uppercase">
+                Hello
+                <span className="absolute bg-brand-4 h-1 left-6 text-brand-4 top-20 w-10" />
```

Comment:
I don't think we should use empty html elements purely for stylistic purposes. This would probably be better as a `before` or `after` element. Check the tailwind [docs](https://tailwindcss.com/docs/adding-new-utilities#using-css) for adding new utilities.

Reply:
@asasmith Sorry I forgot to add aria-hidden="true" to make it semantically correct https://dev.to/wheelmaker24/why-you-don-t-need-every-css-pseudo-selector-in-tailwind-css-3kn1

Reply:
The accessibility part is one thing but it also looks like we can use this bit of styling in 6-7 different components. Moving it to a custom utility class will help keep repetition down.